docs/sources/operators-guide/tools/mimirtool.md

```diff
@@ -872,6 +876,39 @@ printf "{\n%s\n}" "${key_values::-1}"
 
 The only parameter of the script is a file containing the flags, with each flag on its own line.
 
+### Backfill
+
+Backfill command uploads Prometheus TSDB blocks into Grafana Mimir, by using block upload API exposed by compactor component.
```

Comment:
```suggestion
The `backfill` command uploads Prometheus TSDB blocks into Grafana Mimir, by using the block-upload API that is exposed by the compactor component.
```
There are several function words missing from this sentence that I filled in. Please vet for technical accuracy.